[ETTVBridge] New bridge, first push

GregThib opened the pull request:
Bridge for ETTV.tv, with fields to limit the search. Bring some info on files, links, description, etc.

GregThib committed:
[ETTVBridge] New bridge, first push

Bridge for ETTV.tv, with fields to limit the search. Bring some info on files, links, description, etc.

teromene commented on bridges/ETTVBridge.php:
```diff
@@ -0,0 +1,142 @@
+			'type' => 'list',
+			'name' => 'Category',
+			'values' => array(
+				'(ALL TYPES)'				=>  '0',
```
Please put only one space between the `=>` and the value, same for the other lines.

teromene commented on bridges/ETTVBridge.php:
```diff
@@ -0,0 +1,142 @@
+	public function collectData(){
+		// No control on inputs, because all have defaultValue set
+		$query_str  = 'torrents-search.php';
+		$query_str .= '?search='   . urlencode('+'.str_replace(' ',' +',$this->getInput('query')));
```
```$query_str .= '?search='  . urlencode('+' . str_replace(' ', ' +', $this->getInput('query'))); ```

teromene commented on bridges/ETTVBridge.php:
```diff
@@ -0,0 +1,142 @@
+		$query_str .= 'incldead&=' . $this->getInput('status');
+		$query_str .= '&lang='     . $this->getInput('lang');
+		$query_str .= '&sort=id&order=desc';
+		
```
No indentation on empty lines

teromene commented on bridges/ETTVBridge.php:
```diff
@@ -0,0 +1,142 @@
+		// Get results page
+		$html = getSimpleHTMLDOM(self::URI . $query_str)
+			or returnServerError('Could not request ' . $this->getName());
+		
```
No indentation on empty lines

teromene commented on bridges/ETTVBridge.php:
```diff
@@ -0,0 +1,142 @@
+		foreach($html->find('table.table tr') as $element) {
+			if($element->parent->tag == 'thead') continue;
+			$entry = $element->find('td', 1)->find('a', 0);
+			
```
No indentation on empty lines

teromene commented on bridges/ETTVBridge.php:
```diff
@@ -0,0 +1,142 @@
+			$entry = $element->find('td', 1)->find('a', 0);
+			
+			// retrieve result page to get more details
+			$link = rtrim(self::URI,"/") . $entry->href;
```
```$link = rtrim(self::URI, "/") . $entry->href;```

teromene commented on bridges/ETTVBridge.php:
```diff
@@ -0,0 +1,142 @@
+			$link = rtrim(self::URI,"/") . $entry->href;
+			$page = getSimpleHTMLDOM($link)
+				or returnServerError('Could not request page ' . $link);
+			
```
No indentation on empty lines

teromene commented on bridges/ETTVBridge.php:
```diff
@@ -0,0 +1,142 @@
+			// get details & download links
+			$details = $page->find('fieldset.download table', 0); // WHAT?? It should be the second one…
+			$dllinks = $page->find('div#downloadbox table', 0);
+			
```
No indentation on empty lines

teromene reviewed (changes requested):
Thanks for the PR !
Just a few style corrections before it can be merged !

GregThib committed:
[ETTV Bridge] Improved style (mainly indentation)

[ETTV Bridge] Improved style (mainly indentation and spaces) as requested by PR reviewer & CI Test.

GregThib committed:
[ETTV Bridge] Improved spaces to comply Travis

GregThib commented:
Done.

Nota: I do not understand why a test failed in Travis (but, believing the log, the root cause is on the Travis side).
R'lly funny tool by the way. I started using Github recently and I did not know all these tools or processes.

teromene commented:
The reason the test failed on travis is because of the code style checking, that verifies that your codes is valid in regard of our [coding style policy](https://github.com/RSS-Bridge/rss-bridge/blob/master/CONTRIBUTING.md).
You can check manually by calling ```phpcs . --standard=phpcs.xml --warning-severity=0 --extensions=php -p``` on your machine.
Thank you for the bridge, merged !
